test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/reproduce/EnvEndpointKeysConfigTest.kt

```diff
@@ -0,0 +1,151 @@
+package io.micronaut.reproduce
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.micronaut.context.ApplicationContext
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.client.HttpClient
+import io.micronaut.runtime.server.EmbeddedServer
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class EnvEndpointKeysConfigTest {
```

Comment:
tests in `EnvironmentEndpointSpec` already test this functionality in a better way, so this test is largely useless